types.toml

```diff
@@ -14,7 +14,6 @@ start = { date = "2024-01-08T10:00:00.00", timezone = "America/New_York" }
 end = { date = "2024-01-08T11:00:00.00", timezone = "America/New_York" }
 status = "confirmed"
 organizer = { name = "t-types", email = "types@rust-lang.org" }
-recurrence_rules = [ { frequency = "weekly" } ]
```

Comment:
Per [How do I remove an event](https://github.com/rust-lang/calendar?tab=readme-ov-file#how-do-i-remove-an-event), this should probably be:
```toml
recurrence_rules = [ { frequency = "weekly", until = "2024-11-13T00:00:00.00Z" } ]
```

Reply:
oh thank you @Urgau I had forgot about that :-)